### PR TITLE
Undocumented parameters in dispersion

### PR DESCRIPTION
**Pull request by Qudirah**
fixes for #6111

### CODE REVIEW

**Comment by Qudirah** [comment]
@rossbar What do you think?

File: networkx/algorithms/centrality/dispersion.py [hunk 1]
```diff
@@ -19,6 +19,9 @@ def dispersion(G, u=None, v=None, normalized=True, alpha=1.0, b=0.0, c=0.0):
         The target of the dispersion score if specified.
     normalized : bool
         If True (default) normalize by the embededness of the nodes (u and v).
+    alpha, b, c : float
+        If normalized is True (default), try out different values of alpha, b and, c to obtain maximum
+        performance of normalized dispersion.
```
Comment (rossbar) [inline networkx/algorithms/centrality/dispersion.py:24]: In this case I think the best thing to do would be to explicitly describe how the parameters are used in the embeddedness normalization.

**Review by rossbar — CHANGES_REQUESTED** [review]

File: networkx/algorithms/centrality/dispersion.py (hunk 1 above)
Comment (Qudirah) [reply networkx/algorithms/centrality/dispersion.py:24]: " To strengthen the performance of a normalized dispersion, the nodes can be ranked by the order of the function, (disp(u, v) + b)α/(emb(u, v) + c). Searching over choices of alpha, b, and, c leads to maximum performance." How about this?

**Review by rossbar — APPROVED** [review]
Thanks for iterating on this @Qudirah , LGTM!

**Comment by MridulS** [comment]
Thanks @Qudirah!

File: networkx/algorithms/centrality/dispersion.py (hunk 1 above)
Comment (Qudirah) [reply networkx/algorithms/centrality/dispersion.py]: Great. Thanks for your help!

**Comment by Qudirah** [comment]
> Thanks @Qudirah!

:)

**Pull request merged**